src/lib/isEmojy.js

```diff
@@ -0,0 +1,7 @@
+import assertString from './util/assertString';
+
+export default function isEmojy(str) {
+  assertString(str);
+  const regexExp = /(\u00a9|\u00ae|[\u2000-\u3300]|\ud83c[\ud000-\udfff]|\ud83d[\ud000-\udfff]|\ud83e[\ud000-\udfff])/gi;
```

Comment:
It's broken expression.
<img width="178" alt="image" src="https://user-images.githubusercontent.com/9509286/168766101-5f69692d-233e-4d40-bb74-5177a9d223de.png">
It matches Korean character.  #1968